### PR TITLE
Added team page.Issue no.(#7)

### PR DESCRIPTION
**Pull request by Anupam-dagar**
This PR is for the issue #7 

https://anupam-dagar.github.io/humblefoolcup/team.html
Added team page.
I am not able to make header and footer responsive.

Note - After this PR gets accepted, please add your contribution to scores.yml . Refer to README for more details.

### CODE REVIEW

**Comment by Anupam-dagar** [comment]
why header looks different on pc and live link....
screenshot from 2017-01-31 17-42-36
screenshot from 2017-01-31 17-42-44

**Comment by pradeepgangwar** [comment]
That "in memory Harsha Suryanarayana" of line remains fixed and moves with page.

Although it is not so  visible but can be clearly visualize through careful observation.
Make it appear only when page is scrolled down not like one it is now.

Make this change @Anupam-dagar

**Comment by Anupam-dagar** [comment]
I know about that, i tried, i need help regarding these.

**Comment by pradeepgangwar** [comment]
i will see code and reply if i have any help to offer :)

File: team.html [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0" />
+	<link rel="stylesheet" type="text/css" href="css/team.css">
+
+</head>
+
+<body>
+<div id="header">
+  <a href="/" id="logo">HumblefoolCup</a>
+  <ul id="menu">
+    <li><a href="/"><span>Home</span></a></li>
```
Comment (gauthamzz) [inline team.html]: Please provide links

File: css/team.css [hunk 2]
```diff
@@ -0,0 +1,306 @@
+@import 'https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css';
```
Comment (gauthamzz) [inline css/team.css:1]: download these and keep them in css folder

**Review by gauthamzz — CHANGES_REQUESTED** [review]
@magician03 review this for unresponsiveness

**Comment by gauthamzz** [comment]
@Anupam-dagar we need this done ASAP , please give me update

**Comment by Anupam-dagar** [comment]
I will make it responsive,
Do i need to change the design also?
I will make sure next update get accepted.
I will complete it by today.

**Comment by gauthamzz** [comment]
remove the header and menu from team page

**Comment by Anupam-dagar** [comment]
https://anupam-dagar.github.io/humblefoolcup/team.html
Will this work.
screenshot from 2017-02-13 20-35-42

**Comment by Anupam-dagar** [comment]
screenshot from 2017-02-13 21-01-57
Done.

**Pull request merged**